Optional combined chapters

daddy-cool opened the pull request:
GHI (If applicable): N/A

## Description:
Added option to toggle whether to include virtual chapters when real chapters are present or not.

Included native german translation, other translations I would need to generate if required.

## Checklist:
- [x] I have based this PR against the develop branch

pannal commented:
Nice. Will review, but on first look seems great, thanks!

pannal commented:
So you're basically disabling combined chapters when this setting is disabled, right? Which makes virtual chapters optional.

pannal commented:
I don't think I fully agree on the description/name of the setting. Unless I'm getting it wrong.

daddy-cool committed:
Player/SeekDialog/CombineChapters: Added option to toggle whether to combine virtual chapters with video chapters when both are present

daddy-cool commented:
I changed the naming & description to be clearer/easier to understand.
Actual behaviour is unchanged, also added some entries to the .gitignore that I'm otherwise having issues with developing on Windows with VSCode.

daddy-cool commented:
rebased to 1.0.5e

pannal commented:
I'll take a look at this again soon!

pannal commented:
Thanks!
